docs/admin/garbage-collection.md

```diff
@@ -42,15 +42,34 @@ to free. Default is 80%.
 We also allow users to customize garbage collection policy through the following kubelet flags:
 
 1. `minimum-container-ttl-duration`, minimum age for a finished container before it is
-garbage collected. Default is 1 minute.
+garbage collected. Default is 0 minute, means any finshed container will be GC.
```

Comment:
Small typo and some suggestions for grammar and clarity: "Default is 0 minute**s**, **which** means any **finished** container will be **garbaged collected**."